uninstall/3.1/README.md

```diff
@@ -0,0 +1,37 @@
+# Uninstall Cloud Pak for Watson AIOps
+
+The scripts provided here can be used to uninstall Cloud Pak of Watson AIOps and delete resources created by it.
+
+## Prereqs
+- You need to have OC CLI installed
+- You have logged into your cluster using `oc login`
+
+## Getting started
+
+Clone this repo.
+```
+  git clone https://github.ibm.com/katamari/katamari-util.git 
+  cd katamari-util/uninstall/<version to uninstall>
+```
+
+## Preparing for uninstall
+
+There are resources created by Cloud Pak of Watson AIOps that you should review before deleting.  These include PVCs, secrets, CRDs, and config maps.  You might want to save some of these or consider not deleting them if you plan to reinstall.  You can check the `uninstall-cp4waiops-resource-groups.sh` file for a list of resources that the script will delete.  You can update this list with resources you don't want to delete.
+
+There are also resources that are shared by multiple Cloud Paks that Cloud Pak of Watson AIOps might install if it was not already created by a differnt Cloud Pak.  You want to make sure those are not deleted if you are using other Cloud Paks on your cluster.  
+
+Once you have carefully reviewed and decided on what to delete, you can move forward to the next step and update the `uninstall-cp4waiops-props.sh` file with your preferences.
```

Comment:
@kvijai82 : same for uninstall-cp4waiops-props.sh, should we make it as link [uninstall-cp4waiops-props.sh](uninstall-cp4waiops-props.sh) ?

Reply:
Updated.